org.libredirect.frontends-manager.yml

```diff
@@ -0,0 +1,52 @@
+id: org.libredirect.frontends-manager
+runtime: org.gnome.Platform
+runtime-version: '42'
+command: libredirect-frontends-manager
+rename-icon: libredirect-frontends-manager
+
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+  - org.freedesktop.Sdk.Extension.rust-nightly
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --filesystem=xdg-data
```

Comment:
why do you need access to this. It's not allowed by default and usually unecessary.

Reply:
I have asked again about this.